### PR TITLE
Show Ack/Downtime icons in the node tiles for the handled nodes

### PR DESCRIPTION
**Pull request by raviks789**
In case the hosts/services are acknowledged or have scheduled downtime show Acknowledged or
Downtime icons respectively.

ref #237

### CODE REVIEW

**Review by flourish86 — CHANGES_REQUESTED** [review]
Please add a right margin to the `.actions` container

**Review by flourish86 — DISMISSED** [review]
Can you also please remove the right margin in tree view. Thanks!
Screen Shot 2022-08-25 at 10 13 00

**Comment by raviks789** [comment]
> Can you also please remove the right margin in tree view. Thanks!

This change will be performed in a new PR as this is not related to the issue in question (ref #237).

File: public/css/module.less [hunk 1]
```diff
@@ -490,6 +490,15 @@ td > a > .state-badges {
             width: 1.5em;
             height: 1.5em;
             border-radius: 0.3em;
+
+            &.handled-icon {
+                float: right;
+                pointer-events: none;
```
Comment (nilmerg) [inline public/css/module.less:496]: If this wouldn't be an `a`, ignoring pointer events isn't required. Why not just add this class to the `i`?

File: public/css/module.less [hunk 2]
```diff
@@ -490,6 +490,15 @@ td > a > .state-badges {
...
+
+                &:first-child {
```
Comment (nilmerg) [inline public/css/module.less:498]: When is this the case?

**Review by nilmerg — CHANGES_REQUESTED** [review]
The tiles are fine now. Though, we also decided to only show a single icon in the tree view. That's still missing.

**Comment by raviks789** [comment]
> The tiles are fine now. Though, we also decided to only show a single icon in the tree view. That's still missing.

This is implemented in #352.

File: library/Businessprocess/Renderer/TileRenderer/NodeTile.php [hunk 3]
```diff
@@ -259,6 +259,15 @@ protected function addDetailsActions()
...
         }
+
+        if ($node->isAcknowledged()) {
+            $this->actions()->add(Html::tag('i', ['class' => 'icon icon-ok handled-icon']));
+            return;
+        }
+
+        if ($node->isInDowntime()) {
+            $this->actions()->add(Html::tag('i', ['class' => 'icon icon-plug handled-icon']));
+        }
```
Comment (sukhwinder33445) [inline library/Businessprocess/Renderer/TileRenderer/NodeTile.php:270]: - Please use `ipl/web/Icon`.
- Merge the conditions.

**Review by sukhwinder33445 — CHANGES_REQUESTED** [review]

**Review by sukhwinder33445 — APPROVED** [review]

**Review by nilmerg — APPROVED** [review]

**Pull request merged**